AwsCryptographicMaterialProviders/runtimes/java/src/testExamples/java/software/amazon/cryptography/example/hierarchy/mutations/MutationKmsAccessTerminalInFlightTest.java

```diff
@@ -53,7 +54,12 @@ public void test() {
       .build();
     final String branchKeyId =
       testPrefix + java.util.UUID.randomUUID().toString();
-    CreateKeyExample.CreateKey(POSTAL_HORN_KEY_ARN, branchKeyId, null);
+    CreateKeyExample.CreateKey(
+      POSTAL_HORN_KEY_ARN,
```

Comment:
(not blocking) Consider not calling it postal horn?

Reply:
@josecorella tried to stop me from calling the KMS Key ARN 📯 a long time ago,
but I convinced him to let me be happy.

Personally, I think this name is better than something like KMS_KEY_2.